format/FormatNexusEigerDLS16M.py

```diff
@@ -1,46 +1,20 @@
 from __future__ import absolute_import, division, print_function
 
-import ast
-import sys
-
 import h5py
 
 import libtbx
 
-from dxtbx.format.FormatNexus import FormatNexus
+from dxtbx.format.FormatNexusEigerDLS import FormatNexusEigerDLS
 from dxtbx.masking import GoniometerMaskerFactory
 from dxtbx.model import MultiAxisGoniometer
 
 
-def get_count_limit_from_meta(meta_file_name):
-    with h5py.File(meta_file_name, "r") as f:
-
-        config = f["/config"][()]
-        config_data = ast.literal_eval(config.decode("utf-8"))
-
-    return config_data["countrate_correction_count_cutoff"]
-
-
-class FormatNexusEigerDLS16M(FormatNexus):
+class FormatNexusEigerDLS16M(FormatNexusEigerDLS):
     @staticmethod
     def understand(image_file):
-        """Check to see if this format class can understand the image file.
-
-        Args:
-          image_file (str): The file path of the image file to check.
-
-        Returns:
-          bool: Returns ``True`` if the image_file is understood by this format class,
-          else returns ``False``.
-
-        """
```

Comment:
This helpstring is probably still valid for the modified class (how _useful_ it is is another question).

Reply:
Indeed, was removed due to not really containing unique signal (this is the signature for _all_ understand methods)